tests/tests/busola-create-a-deployment.spec.js

```diff
@@ -0,0 +1,162 @@
+      .click({ force: true }) //we need to use force when others elements make menu not visible
+      .wait(1000);
+
+    cy.getIframeBody()
+      .find('[aria-label="open-search"]')
+      .click({ force: true });
+
+    cy.getIframeBody()
+      .find('[placeholder="Search"]')
+      .type(NAMESPACE_NAME);
+
+    cy.getIframeBody()
+      .find('[aria-label="Delete"]')
+      .click({ force: true });
+
+    cy.getIframeBody()
+      .find('[role="status"]')
+      .should('have.text', 'TERMINATING');
+  });
+
+  it('Create a new namespace', () => {
+    getLeftNav()
+      .contains('Namespaces')
+      .click();
+
+    cy.getIframeBody()
+      .contains('Create Namespace')
+      .click();
+
+    cy.getIframeBody()
+      .find('[role=dialog]')
+      .find("input[placeholder='Namespace name']")
+      .should('be.visible')
+      .type(NAMESPACE_NAME);
+
+    cy.getIframeBody()
+      .find('[role=dialog]')
+      .contains('button', 'Create')
+      .click();
+  });
+
+  it('Go to the details of namespace', () => {
+    cy.getIframeBody()
+      .contains('a', NAMESPACE_NAME)
+      .click();
+  });
+
+  it('Create a Deployment', () => {
+    cy.getIframeBody()
+      .contains('Deploy new workload')
+      .click();
+
+    cy.getIframeBody()
+      .find('[role="menuitem"]')
+      .contains('Create Deployment')
+      .click();
+
+    cy.getIframeBody()
+      .find('[placeholder="Deployment name"]')
+      .clear()
+      .type(DEPLOYMENT_NAME);
+
+    cy.getIframeBody()
+      .find('[placeholder="Enter Labels key=value"]')
+      .type(`app=${DEPLOYMENT_NAME}`);
+
+    cy.getIframeBody()
+      .contains('label', 'Labels')
+      .click();
+
+    cy.getIframeBody()
+      .find('[placeholder="Enter Labels key=value"]')
+      .type(`example=${DEPLOYMENT_NAME}`);
+
+    cy.getIframeBody()
+      .contains('label', 'Labels')
+      .click();
+
+    cy.getIframeBody()
+      .find('[placeholder="Enter Docker image"]')
+      .type(DOCKER_IMAGE);
+
+    cy.getIframeBody()
+      .contains('label', 'Memory requests')
+      .next('input')
+      .clear()
+      .type('16Mi');
+
+    cy.getIframeBody()
+      .contains('label', 'Memory limits')
+      .next('input')
+      .clear()
+      .type('32Mi');
+
+    cy.getIframeBody()
+      .contains('label', 'CPU requests')
+      .next('input')
+      .clear()
+      .type('10m');
+
+    cy.getIframeBody()
+      .contains('label', 'CPU limits')
+      .next('input')
+      .clear()
+      .type('20m');
+
+    cy.getIframeBody()
+      .contains('button', 'Create')
+      .click({ force: true });
+  });
+
+  it('Check if deployment and service exist', () => {
+    cy.getIframeBody()
+      .contains('a', DEPLOYMENT_NAME)
+      .should('be.visible');
+
+    getLeftNav()
+      .contains('Discovery and Network')
+      .click({ force: true });
+
+    getLeftNav()
+      .contains('Services')
+      .click({ force: true })
+      .wait(2000);
```

Comment:
You could use `get` or `find` below instead. Cypress will retry looking for the desired element.